lib/src/Pos/Launcher/Launcher.hs

```diff
@@ -63,6 +66,13 @@ launchNode nArgs cArgs lArgs action = do
             (cnaDumpConfiguration cArgs)
             confOpts
 
+    case ekgParams cArgs of
+      Just (EkgParams eHost ePort) -> do
+        putText $ "EKG is starting on " <> show eHost <> ":" <> show ePort
+        _ <- forkServer eHost ePort
+        pure ()
+      Nothing -> pure ()
```

Comment:
Thanks @deepfire, this is better place for it :+1: 

Side-note: `universum` defines a helper `whenJust` for exactly this kind of case:

```hs
whenJust cArgs $ \(EkgParams eHost ePort) -> do
  putText $ "EKG is starting on " <> show eHost <> ":" <> show ePort
  void $ forkServer eHost ePort
```

Reply:
Done!